homeassistant/components/ozw/cover.py

```diff
@@ -74,3 +82,40 @@ async def async_open_cover(self, **kwargs):
     async def async_close_cover(self, **kwargs):
         """Close cover."""
         self.values.primary.send_value(0)
+
+
+class ZwaveGarageDoorBarrier(ZWaveDeviceEntity, CoverEntity):
+    """Representation of a barrier operator Zwave garage door device."""
+
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        return SUPPORT_GARAGE
+
+    @property
+    def device_class(self):
+        """Return the class of this device, from component DEVICE_CLASSES."""
+        return "garage"
```

Comment:
Please import the device class constant from the cover integration and use here.